task/task.go

```diff
@@ -13,11 +13,12 @@ import (
 var ErrTimedOut = errors.New("timed out performing task")
 
 // DoRetryWithTimeout performs given task with given timeout and timeBeforeRetry
-func DoRetryWithTimeout(t func() (interface{}, error), timeout, timeBeforeRetry time.Duration) (interface{}, error) {
+func DoRetryWithTimeout(t func() (interface{}, bool, error), timeout, timeBeforeRetry time.Duration) (interface{}, error) {
```

Comment:
I remember Zoran was using this method somewhere other than stork and torpedo; and probably a couple of places in torpedo as well.
My recommendation is, if possible have a separate method wrapping the original one with the retry boolean. That way we don't have change the original consumers when they upgrade to the latest sched-ops. Else we just keep a different method altogether to avoid changing this in multiple projects.

Reply:
Up to you. Just communicate to Zoran, and probably change in torpedo because torpedo used sched-ops master as of now.